ui/_tests_/utils/testPascletoKebabCase_spec.test.js

```diff
@@ -0,0 +1,8 @@
+import PascalCaseToKebab from "../../utils/PascalCaseToKebab";
+
+describe('PascalKebab String Utility', () => {
+  test('make pasclacase to kebabcase', () => {
```

Comment:
Nitpik: I guess in it will be `pascalcase` and not `pasclacase` 😄 🤔  
Also, if it is so, can you please correct the file name as well? It is currently `testPascletoKebabCase`